FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -188,19 +189,27 @@ extension ModelInfoRetriever {
         dataTask.resume()
       /// FIS token error.
       case .failure:
-        completion(.failure(.internalError(description: ModelInfoRetriever.tokenErrorDescription)))
+        completion(.failure(.internalError(description: ModelInfoRetriever.ErrorDescription
+            .tokenError)))
         return
       }
     }
   }
 
-  /// Date formatter to parse download URL expiry time.
-  private static func getDateFormatter() -> DateFormatter {
-    let dateFormatter = DateFormatter()
-    dateFormatter.locale = Locale(identifier: "en-US_POSIX")
-    dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
-    dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)
-    return dateFormatter
+  /// Parse date from string - used to get download URL expiry time.
+  private static func getDateFromString(_ strDate: String) -> Date? {
+    if #available(iOS 11, *) {
```

Comment:
Nit: also please check which other OS's this is available on, specific versions of macOS, tvOS, watchOS

Reply:
Good catch, done!